op-batcher/flags/flags.go

```diff
@@ -138,6 +152,7 @@ var requiredFlags = []cli.Flag{
 }
 
 var optionalFlags = []cli.Flag{
+	CheckRecentTxsDepthFlag,
```

Comment:
forgot
```suggestion
	WaitNodeSyncFlag
	CheckRecentTxsDepthFlag,
```

Reply:
Fixed in https://github.com/ethereum-optimism/optimism/pull/10193/commits/173c91a18a50c3abeaf8fb6cfa77000dfcc0d644